components/tyk-dashboard/templates/secrets-mounts.yaml

```diff
@@ -0,0 +1,16 @@
+{{- range $secret := .Values.mounts }}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ $.Release.Name }}-dashboard-secret-{{ $secret.name }}
+  labels:
+    app: dashboard-secret-{{ $secret.name }}-{{ include "tyk-dashboard.fullname" $ }}
+    chart: {{ include "tyk-dashboard.chart" $ }}
+    release: {{ $.Release.Name }}
+    heritage: {{ $.Release.Service }}
+type: Opaque
+data:
+  {{ $secret.filename }}: |-
+{{ $.Files.Get (printf "mounts/%s" $secret.filename) | b64enc | indent 4 }}
```

Comment:
Maybe, it'd be good to add `mounts/` folder in the component chart with a README describing how to mount files.